cfpb/design-manual
# Update most components to v10.2.1

This PR updates most Capital Framework components to v10.2.1. Expandables and Tables are left at their previous versions, because updating them breaks this repo's JavaScript compilation, and there are no actual changes they are missing.

Closes #605 
Closes #603


## Additions

- Adds -10 level colors to palette

## Changes

- Updates link hover color
- Corrects link & button colors in page content

## Testing

1. Pull branch
1. `npm install`
1. `grunt`
1. `bundle exec jekyll serve`
1. Visit http://localhost:4000/design-manual/page-components/links.html and confirm that link hover color is now Dark Pacific
1. Visit http://localhost:4000/design-manual/brand-guidelines/color-principles.html to see the new -10 colors in the palette


## Checklist
